src/components/aeBanner/aeBanner.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="ae-banner">
+    <div class="inner">
+      <slot />
+      <slot name="right">
+        <ae-button
```

Comment:
That should be a optional button. also i think the parent component should provide it via slot. there are many designs without a `cancel` button.

Reply:
ok this was in tune with the old notification component.